schemas/user_request.py

```diff
@@ -50,10 +51,9 @@ class CancelRequestIn(BaseModel):
     status_id: int
 
     @validator('status_id')
-    def validate_status_id(cls, v):
+    def validate_status_id(cls, v):  # TODO may be there is need to use classmethod decorator
         if v != 4:
-            message = "Заяву можливо тільки скасувати."
-            raise ValueError(message)
+            raise BackendException(message="The application can only be canceled.")
```

Comment:
Don't use BackendException inside Pydantic schemas validation. it should be handled by another handler: validation_exception_handler.